### PR TITLE
crispy-doom: new package at version 5.11.1

### PR DESCRIPTION
**Pull request by tslilc**
Please fill out the template.

Read: https://k1sslinux.org/wiki/kiss/style-guide

Please include the manifest for version bumps and new packages.

---

## Description of package

From upstream [0]:
> Crispy Doom is a limit-removing enhanced-resolution Doom source port based on [Chocolate Doom](https://www.chocolate-doom.org/wiki/index.php/Chocolate_Doom). Its name means that its internal 640x400 resolution looks "crisp" and is also a slight [reference](http://www.mathsisfun.com/recipie.html) to its origin.

[0] [https://www.chocolate-doom.org/wiki/index.php/Crispy_Doom](https://www.chocolate-doom.org/wiki/index.php/Crispy_Doom)

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/crispy-doom/version
/var/db/kiss/installed/crispy-doom/sources
/var/db/kiss/installed/crispy-doom/manifest
/var/db/kiss/installed/crispy-doom/depends
/var/db/kiss/installed/crispy-doom/checksums
/var/db/kiss/installed/crispy-doom/build
/var/db/kiss/installed/crispy-doom/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/metainfo/io.github.fabiangreffrath.Hexen.metainfo.xml
/usr/share/metainfo/io.github.fabiangreffrath.Heretic.metainfo.xml
/usr/share/metainfo/io.github.fabiangreffrath.Doom.metainfo.xml
/usr/share/metainfo/
/usr/share/man/man6/crispy-hexen.6
/usr/share/man/man6/crispy-hexen-setup.6
/usr/share/man/man6/crispy-heretic.6
/usr/share/man/man6/crispy-heretic-setup.6
/usr/share/man/man6/crispy-doom.6
/usr/share/man/man6/crispy-doom-setup.6
/usr/share/man/man6/
/usr/share/man/man5/hexen.cfg.5
/usr/share/man/man5/heretic.cfg.5
/usr/share/man/man5/default.cfg.5
/usr/share/man/man5/crispy-hexen.cfg.5
/usr/share/man/man5/crispy-heretic.cfg.5
/usr/share/man/man5/crispy-doom.cfg.5
/usr/share/man/man5/
/usr/share/man/
/usr/share/icons/hicolor/128x128/apps/crispy-setup.png
/usr/share/icons/hicolor/128x128/apps/crispy-doom.png
/usr/share/icons/hicolor/128x128/apps/
/usr/share/icons/hicolor/128x128/
/usr/share/icons/hicolor/
/usr/share/icons/
/usr/share/doc/crispy-hexen/INSTALL.hexen
/usr/share/doc/crispy-hexen/CMDLINE.hexen
/usr/share/doc/crispy-hexen/
/usr/share/doc/crispy-heretic/INSTALL.heretic
/usr/share/doc/crispy-heretic/CMDLINE.heretic
/usr/share/doc/crispy-heretic/
/usr/share/doc/crispy-doom/README.md
/usr/share/doc/crispy-doom/README.Music.md
/usr/share/doc/crispy-doom/PHILOSOPHY.md
/usr/share/doc/crispy-doom/NOT-BUGS.md
/usr/share/doc/crispy-doom/NEWS.md
/usr/share/doc/crispy-doom/INSTALL.doom
/usr/share/doc/crispy-doom/ChangeLog
/usr/share/doc/crispy-doom/COPYING.md
/usr/share/doc/crispy-doom/CMDLINE.doom
/usr/share/doc/crispy-doom/
/usr/share/doc/
/usr/share/bash-completion/completions/crispy-hexen
/usr/share/bash-completion/completions/crispy-heretic
/usr/share/bash-completion/completions/crispy-doom
/usr/share/bash-completion/completions/
/usr/share/bash-completion/
/usr/share/applications/screensavers/io.github.fabiangreffrath.Doom_Screensaver.desktop
/usr/share/applications/screensavers/
/usr/share/applications/io.github.fabiangreffrath.Setup.desktop
/usr/share/applications/io.github.fabiangreffrath.Hexen.desktop
/usr/share/applications/io.github.fabiangreffrath.Heretic.desktop
/usr/share/applications/io.github.fabiangreffrath.Doom.desktop
/usr/share/applications/
/usr/share/
/usr/bin/crispy-server
/usr/bin/crispy-hexen-setup
/usr/bin/crispy-hexen
/usr/bin/crispy-heretic-setup
/usr/bin/crispy-heretic
/usr/bin/crispy-doom-setup
/usr/bin/crispy-doom
/usr/bin/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/crispy-doom/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+autoreconf -fi
```
Comment (aabacchus) [inline community/crispy-doom/build:3]: If upstream doesn't provide tarballs with this step already done, (you may want to ask them to), be sure to list `autoconf make` as a dependency.

Comment (tslilc) [reply community/crispy-doom/build:3]: Thanks, good point! I'll see if i can prod upstream for this.

**Comment by tslilc** [comment]
I have opened an issue upstream about `autoconf`

[https://github.com/fabiangreffrath/crispy-doom/issues/874](https://github.com/fabiangreffrath/crispy-doom/issues/874)

**Comment by tslilc** [comment]
Upstream closed `autoconf make` as `WONTFIX`.

**Pull request merged**